src/pages/UserDashboard.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { actions } from '../state/actions';
+
+
+const UserDashboard = (props) => {
+  const {
+    companyReviews,
+    getCompanyReviews
+  } = props;
```

Comment:
I feel like you can destructure the props directly on Line 6

Reply:
I've even removed them